app-network/lldpd/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=lldpd
+PKGDES="Implementation of IEEE 802.1ab (LLDP)"
```

Comment:
Please be more descriptive, this is a toolkit right?

Reply:
Changed to match https://lldpd.github.io/